content/applications/hr/frontdesk.rst

```diff
@@ -0,0 +1,318 @@
+- :guilabel:`Notify by Discuss`: enable this option to have a *Discuss* application message window
+  open with the person the guest is visiting upon check-in. When enabled, a default message appears
+  for the person the guest is visiting. The *Discuss* application must be installed in order for
+  this option to work. *Discuss* is installed by default when creating an Odoo database, and does
+  not count towards billing. As long as the *Discuss* application is not intentionally uninstalled,
+  this option works.
+
+.. example::
+   The default message format for the :guilabel:`Notify by Discuss` option is: `(Frontdesk Station)
+   Check-In: (Guest Name) (Guest Phone Number) (Organization) to meet (Name of employee).`
+
+   An example of how that might appear in a *Discuss* message is: `Main Lobby Check-In: John Doe
+   (123-555-1234) (Odoo, Inc.) to meet Marc Demo.`
+
+
+.. image:: frontdesk/station-form.png
+   :align: center
+   :alt: Frontdesk station form with all the information filled out.
+
+Side Message tab
+~~~~~~~~~~~~~~~~
+
+Enter any desired text to appear on the station kiosk after a guest has checked in, such as a
+welcome greeting or any necessary instructions. The text appears on the confirmation page, on the
+right side of the screen after a guest has completed the check-in process.
+
+.. _frontdesk/drinks:
+
+Drinks
+------
+
+After all the stations are created, the next step is to configure the drinks on offer, if desired.
+This step is not necessary or required for the *Frontdesk* application to work, and only needs to be
+configured if drinks are offered to guests.
+
+To add a drink option, navigate to :menuselection:`Frontdesk application --> Configuration -->
+Drinks`. Click :guilabel:`New` and a blank drink form loads. Enter the following information on the
+form:
+
+- :guilabel:`Drink Name`: type in the name of the drink option in the field. This field is required.
+- :guilabel:`People to Notify`: use the drop-down and select who will be notified when the drink is
+  selected. Multiple people can be entered in the field. This field is required.
+- :guilabel:`Sequence`: enter a numerical value in this field to indicate where in the list of drink
+  options this specific option appears. The lower the number, the higher on the list the drink
+  appears. For example, entering the number one (1) would place that drink at the top of the list
+  and be the first to appear in sequence.
+- :guilabel:`Picture`: hover over the picture icon to reveal the :guilabel:`✏️ (pencil)` icon, and
+  click on the :guilabel:`✏️ (pencil)` icon. A file navigator window appears. Navigate to the
+  desired image file and select it, then click :guilabel:`Open`. The photo now appears in the
+  picture field, and will be set as the background on the kiosk.
+
+.. image:: frontdesk/espresso.png
+   :align: center
+   :alt: Drink form with the information filled out for an espresso.
+
+Stations
```

Comment:
Do you think since there's already another heading called `Stations` near the top of this doc, that this could (or should) be called something else? Just a suggestion, but if it works better this way feel free to ignore

Reply:
I can call this section the Station dashboard- this is a bit more descriptive.